app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
```

Comment:
- [ ] It would be better if we can run the test successfully without any errors, however, this is what is preventing me from running the test
![Screenshot from 2022-04-09 10-30-04](https://user-images.githubusercontent.com/46388113/162568231-b95d68f4-fae9-4eed-bc8c-7b127a26d816.png)